owncloudApp/src/main/java/com/owncloud/android/ui/activity/WhatsNewActivity.java

```diff
@@ -177,28 +188,29 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
             mItem = getArguments() != null ? (FeatureItem) getArguments().getParcelable("feature") : null;
         }
 
+
         @Nullable
         @Override
         public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                                  @Nullable Bundle savedInstanceState) {
-            View v = inflater.inflate(R.layout.whats_new_element, container, false);
 
-            ImageView iv = v.findViewById(R.id.whatsNewImage);
+
+            ImageView iv = binding_element.whatsNewImage;
             if (mItem.shouldShowImage()) {
                 iv.setImageResource(mItem.getImage());
             }
 
-            TextView tv2 = v.findViewById(R.id.whatsNewTitle);
+            TextView tv2 = binding_element.whatsNewTitle;
             if (mItem.shouldShowTitleText()) {
                 tv2.setText(mItem.getTitleText());
             }
 
-            tv2 = v.findViewById(R.id.whatsNewText);
+            tv2 = binding_element.whatsNewText;
             if (mItem.shouldShowContentText()) {
                 tv2.setText(mItem.getContentText());
             }
 
-            return v;
+            return null;
```

Comment:
Nope 😁. What we expect from the `onCreateView` method is to return a view to show in screen, as we did before when returning the variable `v`, so we cannot return null. You have to continue returning a view, but the difference is that now you have to do it with the binding variables instead of `v`. This can be done like:
```suggestion
            return binding_element.getRoot();
```